src/Microsoft.Identity.Web.TokenAcquisition/DefaultTokenAcquisitionHost.cs

```diff
@@ -42,6 +43,12 @@ public DefaultTokenAcquisitionHost(
 
         public string GetEffectiveAuthenticationScheme(string? authenticationScheme)
         {
+            if (string.Equals(authenticationScheme, "MTLS_POP", StringComparison.OrdinalIgnoreCase))
```

Comment:
Not sure what you expect to do here

the ASP.NET Core authenticationScheme is the name of the options. You should not suppose that there will be named options or a config section named MTLS_POP.